src/main/resources/blockly/js/mcreator_blocks.js

```diff
@@ -445,6 +445,54 @@ Blockly.defineBlocksWithJsonArray([
         "output": "Direction",
         "colour": "20"
     },
+    {
+        "type": "time_to_string",
+        "message0": "Time to text",

```

Comment:
This block provides time in text format, it does not convert time to text as it does not take any input, so I would consider rewording this block's text a bit